src/components/Learn-Components/Setup-Pre-Requisites/index.js

```diff
@@ -13,33 +12,39 @@ const SetupPreReq = () => (
       <p>In this learning path, we will be using Meshery as the management plane to manage the service meshes. Meshery is the service mesh management plane which offers lifecycle, configuration, and performance management of service meshes and their workloads.</p>
     </div>
     <div className="container-card" id="Set up">
-      <Link to="/service-mesh-management/meshery">
+      <a href="/service-mesh-management/meshery" 
+        target = "_blank" 
+        rel = "noopener noreferrer">
```

Comment:
The `rel` attribute needs to be removed.

Reply:
I'll remove it by today. @leecalcote

Reply:
Hey man, I'll remove it by today.
Sorry for the delay.